demos/go/gin/config.env.example

```diff
@@ -0,0 +1,4 @@
+APIKEY= <..APIKEY..>
+APISECRET= <..APISECRET..>
+ACCESS_TOKEN= <..ACCESS_TOKEN..>
```

Comment:
@gagan3012 
we can not have the token in env file. The value of the token will be available after successful authentication.

Reply:
Removed Access_token from env file